website/community/index.mdx

```diff
@@ -72,6 +64,29 @@ Enhancements or new feature proposals are also welcome. The more concrete and ra
 
 * OpenDAL core repository: https://github.com/apache/incubator-opendal
 
+## Upcoming Community Events
+
+For now, we have a single community event known as the Community Sync Meeting.
+This meeting is open to everyone.
+
+During the meeting, we will cover various topics including project status,
+roadmap updates, and other relevant discussions.
+We will also have a Q&A session at the end of the meeting.
+
+Attending the meeting presents a valuable opportunity
+for you to meet fellow community members and gain deeper insights into the project.
+Moreover, it provides an excellent chance for active involvement in the project's activities.
+
+We currently organize the community sync meeting every three weeks.
+The meeting takes place at 2pm on the Friday of the final week in each three-week cycle.
```

Comment:
Perhaps we need to mark the time zone. Also, cc @Xuanwo, I remember we had a discussion about changing the time on the email list. Do we have any results?

Reply:
https://lists.apache.org/thread/nfggny089hswrldjxqngkf4zppn6ph9k

> we adjust the community sync times to two sessions as follows: one at 09:00 UTC+8 and another at 19:00 UTC+8. For example, our upcoming meeting is scheduled for 09:00 UTC+8 on January 5, 2024, and the following meeting will take place at 19:00 UTC+8 on January 26, 2024.

Reply:
> Perhaps we need to mark the time zone.

Oh sorry, forget about the time zone. I'd add the time zone here first.
After the discussion, we can update this.

Reply:
The discussion already closed. Please update with the new time.